### ISSUE TITLE
Node 5.6.0 doesn't extract on Windows

### ISSUE THREAD

**Issue by shawmanz32na** [srs/gradle-node-plugin#96]
As a workaround for https://github.com/srs/gradle-node-plugin/issues/64 , our team has been specifying the use of a specific version of node to download on demand by specifying
```
node {
    download = true
    version = '5.6.0'
}
```
in our build.gradle file.

However, I recently started using the stable branch of Node.js, 5.6.0, and ( https://github.com/srs/gradle-node-plugin/issues/64 is still an issue) I receive 
```
 PS C:\Users\kshaw\Source\soi-admin> ./gradlew build
Download https://nodejs.org/dist/v5.6.0/node-v5.6.0-linux-x86.tar.gz
Download https://nodejs.org/dist/v5.6.0/win-x64/node.exe
.........
:nodeSetup FAILED

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':nodeSetup'.
> Unable to expand TAR 'C:\Users\kshaw\.gradle\caches\modules-2\files-2.1\org.nodejs\node\5.6.0\4aa8406fb88892c05f9aa500f5d800e6e4c8713e\node-5.6.0-linux-x86.tar.gz'
  The tar might be corrupted or it is compressed in an unexpected way.
  By default the tar tree tries to guess the compression based on the file extension.
  If you need to specify the compression explicitly please refer to the DSL reference.

* Try:
Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output.

BUILD FAILED

Total time: 22.546 secs
```

I'm using a current build of Windows 10 x64, Node 5.6.0, Gradle 2.11, gradle-node-plugin 0.11 .

I'm happy to attach logs including the --stacktrace option, if it will help.

**Comment by srs** [comment]
Thanks for reporting. Will look into this.

**Comment by srs** [comment]
I think this is due to some other concurrency issues. Are you running parallell build?

**Comment by srs** [comment]
This could be the same as #91.

**Comment by srs** [comment]
Closing this since it's fixed in 0.12 and above.
